overlord/snapstate/backend_test.go

```diff
@@ -886,7 +886,7 @@ func (f *fakeSnappyBackend) StartServices(svcs []*snap.AppInfo, meter progress.M
 	return nil
 }
 
-func (f *fakeSnappyBackend) StopServices(svcs []*snap.AppInfo, reason snap.ServiceStopReason, meter progress.Meter, tm timings.Measurer) error {
+func (f *fakeSnappyBackend) StopServices(svcs []*snap.AppInfo, flags *backend.StopFlags, reason snap.ServiceStopReason, meter progress.Meter, tm timings.Measurer) error {
 	f.appendOp(&fakeOp{
 		op:   fmt.Sprintf("stop-snap-services:%s", reason),
```

Comment:
not for this PR, but when we start using/passing flags around for real, we should store that in the fakeOp to ensure that the expected flag is passed around.

Reply:
Will do in a followup, thanks for suggestion.